flepimop/gempyor_pkg/src/gempyor/model_info.py

```diff
@@ -234,6 +232,15 @@ def __init__(
         self.seir_modifiers_library = None
         if config["seir"].exists():
             self.seir_config = config["seir"]
+            if not self.seir_config["integration"].exists():
+                self.seir_config["integration"]["method"].set("rk4")
+                self.seir_config["integration"]["dt"].set(2)
+            else:
+                if not self.seir_config["integration"]["method"].exists():
+                    self.seir_config["integration"]["method"].set("rk4")
+                if not self.seir_config["integration"]["dt"].exists():
```

Comment:
dt=2.0 is only supported for rk4 methods I think (or perhaps euler too, but definetly not expected to work for every methods as seeding needs to be evaluated each day). Let me check

Reply:
propose that we deal with this in a new issue that would roughly:
 - change the dt default(s)
 - ensure that non-rk4 methods can "work" for an arbitrary dt (though obviously not necessarily being good numerical approximations) with seeding / etc that may have a defined pace.